backstop_data/deploy_comment.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+
+# Only upload test result when the job is triggered by pull request
```

Comment:
Is it possible to still have a manual trigger that does not require a pull request? Just in case of an accidental merge / overwrite?

Reply:
Yes, you can manual trigger it. I did it all the time while testing this service.
1. First locate the PR check and click the `Details` button.
![image](https://user-images.githubusercontent.com/1207520/75402192-17389680-58b9-11ea-9864-dd02a41941e1.png)
2. After it brings you to the travis-ci page, you can `Restart build`.
![image](https://user-images.githubusercontent.com/1207520/75402298-46e79e80-58b9-11ea-8661-752e8cfe5dca.png)